Random seed: 4348906705673819894
DDPG model not found or failed to load. Using default strategy. Error: [Errno 2] No such file or directory: '/Users/james/Desktop/C1GamesStarterKit-master/python-algo/ddpg_actor_model.pth'
---------------- Starting Your Algo --------------------
Configuring your custom DDPG strategy...
Turn 0 using DDPG strategy
DDPG selected action: [-0.08209267258644104, 0.002593861659988761, -0.0029249051585793495, 0.16128654778003693]
Traceback (most recent call last):
  File "/Users/james/Desktop/C1GamesStarterKit-master/python-algo/algo_strategy_ddpg.py", line 243, in <module>
    algo.start()
  File "/Users/james/Desktop/C1GamesStarterKit-master/python-algo/gamelib/algocore.py", line 74, in start
    self.on_turn(game_state_string)
  File "/Users/james/Desktop/C1GamesStarterKit-master/python-algo/algo_strategy_ddpg.py", line 174, in on_turn
    index_defence = int(round(((defence_vector[1] + 1) / 2) * (len(allowed_positions) - 1)))
IndexError: list index out of range
